files/scripts/syncd.sh

```diff
@@ -75,6 +98,17 @@ function waitplatform() {
 
 function stopplatform1() {
 
+    if [[ x"$(${SONIC_DB_CLI} STATE_DB GET ORCH_ABRT_STATUS)" == x"1" ]]; then
+        # Collecting saisdkdump before restarting syncd
+        # Runs when orchagent is aborted because of SAI failure.
+        # Only enabled for mellanox platform
+        if [[ x$sonic_asic_platform == x"mellanox" ]]; then
+            collect_saisdkdump
+        fi
+        # This is used to notify auto-techsupport process
+        touch /tmp/saidump_collection_notify_flag
```

Comment:
Should we touch only when auto tech support is enabled?

Reply:
Actually no, auto-TS is equipped to delete this file even if the feature is disabled. Besides, It's not clean and rather clumsy to use AUTO_TS config checks in this script